pkg/lib/credentials.js

```diff
@@ -242,6 +252,19 @@ function Keys() {
                 });
 
         return dfd.promise();
+    }
+
+    self.change = function change(name, old_pass, new_pass, two_pass) {
+        return run_keygen(name, null, old_pass, new_pass, two_pass);
+    };
+
+    self.create = function create(name, type, new_pass, two_pass) {
+        return ensure_ssh_directory(name)
+                .then(() => run_keygen(name, type, "", new_pass, two_pass));
```

Comment:
Can you please pass `null` for old_pass, to make sure that we get a proper crash instead of an accidental empty password if the pattern matching does the wrong thing somewhere?